build.rs

```diff
@@ -136,14 +150,25 @@ mod generate {
 
         wrapper.flush()?;
 
-        // Then generate bindings from that file
-        bindgen::builder()
+        let mut builder = bindgen::builder()
             .ctypes_prefix("crate::_fake_ctypes")
             .header(wrapper_path.as_os_str().to_str().unwrap())
```

Comment:
Feel free to clean up this unneeded `.as_os_str()` too - in addition this build script has quite a few more lint smells that could be resolved :grimacing: